builtin/providers/aws/resource_aws_s3_bucket_test.go

```diff
@@ -131,16 +131,19 @@ func testAccCheckAWSS3BucketWebsite(n string, indexDoc string, errorDoc string)
 		}
 
 		if *out.IndexDocument.Suffix != indexDoc {
-			return fmt.Errorf("bad: %s", out.IndexDocument)
+			if out.IndexDocument.Suffix != nil {
+				return fmt.Errorf("bad index document suffix: %s", *out.IndexDocument.Suffix)
+			}
+			return fmt.Errorf("bad index document suffix, is nil")
```

Comment:
_not_ checking for nil when we're reaching deep like that has bitten me in the past. Just being cautious